src/main/java/ReviewResponse.java

```diff
@@ -1,10 +1,13 @@
 import lombok.Data;
 import lombok.Getter;
+import lombok.Setter;
 import lombok.ToString;
 
 import java.util.List;
 
 @ToString
+@Setter
+@Getter
```

Comment:
Using Lombok's @Getter annotation can lead to violating the Encapsulation principle of SOLID by exposing class fields directly. It may hinder future changes to the class. (Severity: medium)